src/main/java/com/teamfive/disscard/DissCardController.java

```diff
@@ -58,54 +66,54 @@ public String index(Model model) {
      * Fetches a list of all cards registered with DissCard
      * @return A list of JSON objects representing cards
      */
-    @GetMapping("/card/list")
+    @GetMapping(value = "/card/list", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Use produces attribute to specify response type as JSON:
Source: https://www.tutorialspoint.com/java/java_class_attributes.htm